### PR TITLE
Enforce rubocop on CI

### PR DESCRIPTION
**Pull request by jcoyne**
Autocorrected what could be autocorrected and generated config todo for the rest

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ before_script:
   - ./cc-test-reporter before-build
 script:
   - bundle exec rake
+  - bundle exec rubocop
```
Comment (mjgiarlo) [inline .travis.yml]: I might rather see rubocop run first so the failure is fast. Or, alternatively, handle this in the Rakefile (like https://github.com/sul-dlss/dlme-transform/blob/master/Rakefile#L7-L14). Don't feel too strongly about it, though.

File: lib/assembly-objectfile/object_fileable.rb [hunk 2]
```diff
@@ -148,7 +150,117 @@ def exif_mimetype
       @exif_mimetype ||= begin
         check_for_file
         prefer_exif = !Assembly::TRUSTED_MIMETYPES.include?(file_mimetype) # if it's not a "trusted" mimetype and there is exif data; get the mimetype from the exif
-        exif.mimetype if exif && exif.mimetype && prefer_exif
+        exif.mimetype if # require 'checksum-tools'
+# Common behaviors we need for other classes in the gem
```
Comment (ndushay) [inline lib/assembly-objectfile/object_fileable.rb]: what is this huge comment block, and why is it in the rubocop commit?  It seems like a separate concept

File: lib/assembly-objectfile/object_fileable.rb [hunk 3]
```diff
@@ -148,7 +150,117 @@ def exif_mimetype
       @exif_mimetype ||= begin
         check_for_file
         prefer_exif = !Assembly::TRUSTED_MIMETYPES.include?(file_mimetype) # if it's not a "trusted" mimetype and there is exif data; get the mimetype from the exif
-        exif.mimetype if exif && exif.mimetype && prefer_exif
+        exif.mimetype if # require 'checksum-tools'
```
Comment (mjgiarlo) [inline lib/assembly-objectfile/object_fileable.rb]: :eyes: What happened here?

**Comment by jcoyne** [comment]
@ndushay I think it's incorrect to block one PR because a different unrelated PR needs to be made.

File: lib/assembly-objectfile/object_fileable.rb (hunk 2 above)
Comment (jcoyne) [reply lib/assembly-objectfile/object_fileable.rb]: Bug in rubocop 0.75.0, seems better when I use 0.75.1

**Comment by mjgiarlo** [comment]
@ndushay is this blocked from being merged due to 
> please can you also add (in a separate PR) the PR template for this repo?

:question:

**Comment by ndushay** [comment]
this repo needs the PR template;  this PR helped me notice it;  I put the template in the app prs, but didn't do the gems;  I was trying to say that since you were already in this PR, would you take the 2 min and put in a separate PR for the templates.

**Pull request merged**